packages/server/src/ratelimit.ts

```diff
@@ -1,21 +1,23 @@
 import { tooManyRequests } from '@medplum/core';
 import { Request } from 'express';
-import rateLimit, { MemoryStore, RateLimitRequestHandler } from 'express-rate-limit';
+import rateLimit, { RateLimitRequestHandler } from 'express-rate-limit';
+import RedisStore from 'rate-limit-redis';
 import { AuthenticatedRequestContext, getRequestContext } from './context';
-
-/*
- * MemoryStore must be shutdown to cleanly stop the server.
- */
+import { getRedis } from './redis';
 
 const DEFAULT_RATE_LIMIT_PER_15_MINUTES = 15 * 60 * 1000; // 1000 requests per second
-const DEFAULT_AUTH_RATE_LIMIT_PER_15_MINUTES = 600;
+const DEFAULT_AUTH_RATE_LIMIT_PER_15_MINUTES = 1200;
 
 let handler: RateLimitRequestHandler | undefined = undefined;
-let store: MemoryStore | undefined = undefined;
+let store: RedisStore | undefined = undefined;
 
 export function getRateLimiter(): RateLimitRequestHandler {
   if (!handler) {
-    store = new MemoryStore();
+    const client = getRedis();
+    store = new RedisStore({
+      // @ts-expect-error - Pass string args direct to ioredis
```

Comment:
A more descriptive error would be useful for future readers
```suggestion
      // See: https://www.npmjs.com/package/rate-limit-redis#:~:text=//%20%40ts%2Dexpect%2Derror%20%2D%20Known%20issue%3A
      // @ts-expect-error - Known issue: the `call` function is not present in @types/ioredis
```

Reply:
K, will do.

"Known issue: the `call` function is not present in @types/ioredis" is no longer correct.  The `call` function is present, it just has a bunch of TypeScript shenanigans for the proper ordering of Redis commands.

Reply:
I see, fair. I guess the readme hasn't been updated yet